jobs/update-devserver-static-images.groovy

```diff
@@ -91,7 +91,7 @@ def publishResults() {
 
 onMaster('10h') {
    notify([slack: [channel: params.SLACK_CHANNEL,
-                   failureChannel: "#infrastructure-platform",
+                   failureChannel: "#local-devserver",
                    sender: 'Devserver Duck',
                    emoji: ':duck:',
                    when: ['SUCCESS', 'FAILURE', 'UNSTABLE', 'ABORTED']]]) {
```

Comment:
questioning if it makes sense to send the success/unstable/aborted messages tho, maybe just failures?